core/src/main/scala/org/apache/spark/status/AppStatusStore.scala

```diff
@@ -36,7 +36,8 @@ import org.apache.spark.util.kvstore.KVStore
  */
 private[spark] class AppStatusStore(
     val store: KVStore,
-    val listener: Option[AppStatusListener] = None) {
+    val listener: Option[AppStatusListener] = None,
+    val storePath: Option[File] = None) {
```

Comment:
A new parameter has been added, which can also be changed into a class field with init value `None`